spec/models/manageiq/providers/microsoft/infra_manager/vm_scan_spec.rb

```diff
@@ -0,0 +1,20 @@
+describe VmScan do
+  context "A single VM Scan Job on Microsoft provider" do
+    let(:vm) do
+      vm = double("ManageIQ::Providers::Microsoft::InfraManager::Vm")
+      allow(vm).to receive(:kind_of?).with(ManageIQ::Providers::Openstack::CloudManager::Vm).and_return(false)
+      allow(vm).to receive(:kind_of?).with(ManageIQ::Providers::Microsoft::InfraManager::Vm).and_return(true)
+      vm
```

Comment:
Can we just create a vm with the `:vm_microsoft` factory that way we don't have to stub `.kind_of?`